sandbox/lua/modules/graphite.lua

```diff
@@ -0,0 +1,135 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
+Module contains utilities for Graphite
+--]]
+-- Imports
+require "string"
+require "math"
+require "table"
+
+local M = {}
+
```

Comment:
This plugin doesn't use the global preservation hack and can be written using the preferred environment method (I was just clarifying the other alert and anomaly must remain the same).